Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1229,6 +1229,13 @@ public extension Purchases {
 #if os(iOS)
     /**
      * Displays a sheet that enables users to redeem subscription offer codes that you generated in App Store Connect.
+     *
+     * - Note: Even though the docs in `SKPaymentQueue.presentCodeRedemptionSheet`
```

Comment:
There's syntax to render this as a warning, can't recall if it's `- Warn:` or `- Warning:` or what it is, but it's more appropriate for this case since it's potentially app-breaking

Reply:
I sent some notes over slack about fixing the react-native issue